README.md

```diff
@@ -1,6 +1,10 @@
-# Code PushUp CLI
+<h1 align="center">Code PushUp CLI</h1>
+<img alt="Code-Pushup Logo" src="./packages/cli/docs/images/logo.png" width="985">
+<h2 align="center">Comprehensive tech quality monitoring</h2>
+<!-- link when web landing is ready <a href="./packages/cli/README.md#getting-started">Try our paid features</a> -->
+<p align="center">Monitor regressions - Quantify tech debt</p>
```

Comment:
:art: I think adding another minor slogan would use the horizontal space better.[^1] Also, an em dash (`—`) instead of regular dash (`-`) works better as a separator here both visually and semantically.

<details><summary> :framed_picture: Screenshot</summary>
<p>

![image](https://github.com/code-pushup/cli/assets/34691111/3ee43754-073d-45c3-8bd7-4b9912decb17)


</p>
</details> 



```suggestion
<p align="center">Quantify tech debt &mdash; Track incremental improvements &mdash; Monitor regressions</p>
```

[^1]: _Track incremental improvements_ was another of my suggetions in https://github.com/code-pushup/cli/pull/718#discussion_r1655067987. I ordered the mini-slogans based on what I think are our best USPs (e.g. _Monitoring regressions_ is least unique).